Composer/packages/client/src/pages/design/DebugPanel/WatchVariablePicker/WatchVariablePicker.tsx

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { useMemo, useCallback, useEffect, useRef, FocusEvent, KeyboardEvent, useState, FormEvent } from 'react';
+import { TextField, ITextField, ITextFieldStyles } from 'office-ui-fabric-react/lib/TextField';
+import debounce from 'lodash/debounce';
+import { IContextualMenuItem, ContextualMenu, DirectionalHint } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { FontSizes, SharedColors } from '@uifabric/fluent-theme';
+import { useRecoilValue } from 'recoil';
+
+import { getDefaultFontSettings } from '../../../../recoilModel/utils/fontUtil';
+import { dispatcherState, rootBotProjectIdSelector, watchedVariablesState } from '../../../../recoilModel';
+
+import { PropertyItem } from './utils/components/PropertyTreeItem';
+import { useNoSearchResultMenuItem } from './utils/hooks/useNoSearchResultMenuItem';
+import { computePropertyItemTree, getAllNodes, WatchDataPayload } from './utils/helpers';
+import { getPickerContextualMenuItem } from './utils/components/PickerContextualMenuItem';
+
+const DEFAULT_FONT_SETTINGS = getDefaultFontSettings();
+
+type WatchVariablePickerProps = {
+  payload: WatchDataPayload;
+  disabled?: boolean;
+  variableId: string;
+  path: string;
+};
+
+const getStrings = () => {
+  return {
+    emptyMessage: formatMessage('No properties found'),
+    searchPlaceholder: formatMessage('Add a property'),
+  };
+};
+
+const textFieldStyles = (errorMessage?: string): Partial<ITextFieldStyles> => ({
+  field: {
+    fontFamily: DEFAULT_FONT_SETTINGS.fontFamily,
```

Comment:
same comment

Reply:
If the user settings font affects the form editors (all inputs in the app) then it should probably use the user settings font, otherwise I think it should use the default because it is a text input and not a code editor